### PR TITLE
Win

### PR DESCRIPTION
**Pull request by daelynj**

### CODE REVIEW

File: lib/poker_hands/game.rb [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
     deck = Deck.new
 
-    hand1 = deck.draw(5)
-    hand2 = deck.draw(5)
+    @hand1 = deck.draw(5)
+    @hand2 = deck.draw(5)
+  end
+
+  def determine_winner
+    CompareHands.new.call(@hand1, @hand2) ? 'Hand 1 wins' : 'Hand 2 wins'
```
Comment (jgayfer) [inline lib/poker_hands/game.rb:13]: Minor nit, we started using `CompareHands` in this commit, but we didn't define where or what it was until the next commit.

We generally we don't want our commits to depend on things that don't exist yet.

File: lib/poker_hands/CompareHands.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class CompareHands
+  def call(hand1, hand2)
+    true
```
Comment (jgayfer) [inline lib/poker_hands/CompareHands.rb:3]: Will this method always return a boolean? (Does it make sense for it to return a boolean?)

**Comment by jgayfer** [comment]
As an additional note, the PR title + description are nice to include. They help reviewers situate themselves in your world and the problem that we're solving.

File: lib/poker_hands/game.rb (hunk 1 above)
Comment (daelynj) [reply lib/poker_hands/game.rb:13]: Would it have been better to commit the CompareHands class and then commit the code that uses it?

File: lib/poker_hands/CompareHands.rb (hunk 2 above)
Comment (daelynj) [reply lib/poker_hands/CompareHands.rb:3]: In the case of the `determine_winner` method above, it returns the winner based on a true or false value, but to answer your question no, when you compare two hands you're not expecting a boolean, if it was `check_if_hand_wins(@hand1)` then you could expect a boolean.

**Review by jgayfer — APPROVED** [review]

**Pull request merged**